TestFoundation/TestNSString.swift

```diff
@@ -1211,22 +1211,41 @@ class TestNSString : XCTestCase {
 }
 
 struct ComparisonTest {
+    enum TestBehavior {
+      case run
+      case xfail(String)
```

Comment:
xfail may be used in other places but is not very readable imho. Can we expand this to perhaps expectedFailure or something more indicative of what it is meant to be used for?

Reply:
Sure, I'll make the change.